### DISCUSSION TITLE
Using dhcp option 121 (249)

### DISCUSSION THREAD

**Discussion by realdomdom** [AdguardTeam/AdGuardHome#6059] [Q&A]
I tried using option 121 in the dhcp section to propagate a default route for 8.8.8.8 like so:

```
dhcp4:
...
options: 
- '121 hex 2008080808C0A8B22A' 
```
but every time I apply those settings AdGuard Home stops responding (to queries on the web interface and dns requests).
Then I thought maybe because it's a Windows client requesting I also need:
```
- '249 hex F9092008080808C0A8B22A'
```
but my problem remains the same.

What am I doing wrong?

(I used this website to generate the value for 121 https://www.medo64.com/2018/01/configuring-classless-static-route-option/ and 249 https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-dhcpe/f9c19c79-1c7f-4746-b555-0c0fc523f3f9)

**Comment by AaronIsMyBrother** [comment]
I would like to ask you a question. Did you try to start AG with not 'root' user?

**Reply by realdomdom** [reply]
I start it with `sudo systemctl status AdGuardHome.service start`.

**Reply by iam-py-test** [reply]
Ignore AaronIsMyBrother, they are a spammer
